src/app/onion/core/learning-object.service.ts

```diff
@@ -494,6 +494,25 @@ export class LearningObjectService {
     return this.handleFileMetaRequests(files, route);
   }
 
+  async changeStatus({
+    username,
+    objectId,
+    status,
+    reason
+  }: {
+    username: string,
+    objectId: string,
+    status: LearningObject.Status,
+    reason?: string
+  }): Promise<void> {
+    await this.http.post<void>(
+      ADMIN_ROUTES.CHANGE_STATUS(username, objectId),
+      { status, reason },
+      // @ts-ignore
```

Comment:
Just curious what error does this suppress?

Reply:
For some reason "response-type" isn't a valid optional parameter anymore.  Without it, if we get a 200 the UI would treat it as an error.